src/commands/migrations/generate/README.md

```diff
@@ -0,0 +1,166 @@
+# Migrations Generate Command
+
+The `storyblok migrations generate` command allows you to generate migration files for specific component fields. This is useful when you need to transform or update field values across your Storyblok content.
+
+## Basic Usage
+
+```bash
+# Generate a migration for a specific component field
+storyblok migrations generate my-component --field my-field --space 295017
```

Comment:
@alvarosabu shouldn't the --field flag be a more specific case rather than the common one? IMO, we can just have 1 of the examples with the --field flag and the rest without it

Reply:
actually `--field` is not an option of the generate command